v1.1/uuid.md

```diff
@@ -0,0 +1,135 @@
+With braces | The standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format with braces.<br><br>Example: {acde070d-8c4c-4f0d-9d8a-162843c10333}
+As Bytes | UUID value specified as bytes.<br><br>Example: b'kafef00ddeadbeed'
+UUID used as a URN | UUID can be used as a Uniform Resource Name (URN). In that case, the format is [specified](https://www.ietf.org/rfc/rfc2141.txt) as "urn:uuid:" followed by standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format.<br><br>Example: 'urn:uuid:63616665-6630-3064-6465-616462656564'
+
+## Size
+A `UUID` value is 128 bits in width, but the total storage size is likely to be larger due to CockroachDB metadata. 
+
+## Examples
+
+### Create a table with manually-entered UUID values
+
+#### Create a table with UUID in standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format.
+
+~~~ sql
+> CREATE TABLE v (token uuid);
+
+> INSERT INTO v VALUES ('63616665-6630-3064-6465-616462656562');
+
+> SELECT * FROM v;
+~~~
+
+~~~
++--------------------------------------+
+|                token                 |
++--------------------------------------+
+| 63616665-6630-3064-6465-616462656562 |
++--------------------------------------+
+(1 row)
+~~~
+
+#### Create a table with UUID in standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format with braces.
+
+~~~ sql
+> INSERT INTO v VALUES ('{63616665-6630-3064-6465-616462656563}');
+
+> SELECT * FROM v;
+~~~
+
+~~~
++--------------------------------------+
+|                token                 |
++--------------------------------------+
+| 63616665-6630-3064-6465-616462656562 |
+| 63616665-6630-3064-6465-616462656563 |
++--------------------------------------+
+(2 rows)
+~~~
+
+#### Create a table with UUID in byte format.
+
+~~~ sql
+> INSERT INTO v VALUES (b'kafef00ddeadbeed');
+
+> SELECT * FROM v;
+~~~
+
+~~~
++--------------------------------------+
+|                token                 |
++--------------------------------------+
+| 63616665-6630-3064-6465-616462656562 |
+| 63616665-6630-3064-6465-616462656563 |
+| 6b616665-6630-3064-6465-616462656564 |
++--------------------------------------+
+(3 rows)
+~~~
+
+#### Create a table with UUID used as URN.
+
+~~~ sql
+> INSERT INTO v VALUES ('urn:uuid:63616665-6630-3064-6465-616462656564');
+
+> SELECT * FROM v;
+~~~
+
+~~~
++--------------------------------------+
+|                token                 |
++--------------------------------------+
+| 63616665-6630-3064-6465-616462656562 |
+| 63616665-6630-3064-6465-616462656563 |
+| 6b616665-6630-3064-6465-616462656564 |
+| 63616665-6630-3064-6465-616462656564 |
++--------------------------------------+
+(4 rows)
+~~~
+
+### Create a table with auto-generated unique row IDs
+
+To auto-generate unique row IDs, you can cast `UUID` as bytes and use the `uuid_v4()` function. 
```

Comment:
Let's use the same language we use in the FAQ:

> To auto-generate unique row IDs, use the `UUID` column with the `uuid_v4()` function cast to UUID as the default value:

Alternatively, we could just use the same include here that we do elsewhere and safe some maintenance effort.